LabTools/DataManagement.py

```diff
@@ -335,16 +338,16 @@ def displayer(self, data):
             print("displayer triggered")
 
 
-class DataWriter(QObject):
+class DataWriter(QObject): # possibly unused?
```

Comment:
Yes it hasn't been used in the end, we could delete it. With PyCharm, one can use a `refactor` function, which looks for all occurence of a function in the code before one deletes or modifies it.

Reply:
According to PyCharm refactor, this was infact unused and subsequently removed